cluster/gce/trusty/configure.sh

```diff
@@ -45,8 +45,10 @@ config_ip_firewall() {
 create_dirs() {
   # Create required directories.
   mkdir -p /var/lib/kubelet
-  mkdir -p /var/lib/kube-proxy
   mkdir -p /etc/kubernetes/manifests
+  if [ "${KUBERNETES_MASTER:-}" = "false" ]; then
```

Comment:
This file is sourced and executed in upstart jobs. Unfortunately, upstart does not recognize "[[". It fails with message like "/proc/self/fd/9: 52: [[: not found". But it works with single '['. This is why all shell code under cluster/gce/trusty uses '[' instead of "[["